scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -13,19 +13,26 @@
             <div class="row align-items-between align-items-center">
               <div class="col-10">
                 <i class="fa fa-check-circle-o"></i>
-		            {% if variant.category == "snv" %}
+		            {% if variant.category in ("snv", "cancer", "str") %}
                   <a href="{{ url_for('variant.variant',
```

Comment:
Wait a moment' won't this link be wrong if the variant is not a NON-cancer snv? then?

Reply:
Hm, no?? The same variant page for all of the three, and same sv page for sv and cancer_sv?

Reply:
Yes, right, sorry! 👍